cpp/data/sql_updates/ub_tools.23

```diff
@@ -0,0 +1 @@
+ALTER TABLE metadata_presence_tracer ADD COLUMN marc_subfield_code CHAR(1) AFTER marc_field_tag;
```

Comment:
should be the same as ub_tools.sql (e.g. NOT NULL missing)

Reply:
Done